### PR TITLE
Update UG for SeedData, Clear commands

### PR DESCRIPTION
**Pull request by kokerinks**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -176,18 +179,18 @@ Examples:
...
+  * Members added by users
   * Seed data added from `seedData` command
-* User will be prompted to verify the clear command, before carrying out the clearing.
+* User will be prompted to verify the clear command.
 * To bypass the verification prompt, the user can follow `clear` with `--force`. This will clear the address book without additional prompting.
 
 ### Adding points: `addpoints`
 
-Adds points for a member in the loyalty program.
+Add points for a member in the loyalty program.
```
Comment (billyhoce) [inline docs/UserGuide.md:193]: I think "Adds" is grammatically correct here:

the 'addpoints' command **adds** points for a member.....

**Review by billyhoce — COMMENTED** [review]
Thanks for fixing the inconsistent formatting! A small change and i think its good to go!

**Comment by billyhoce** [comment]
LGTM!

**Comment by kinjalagarwal1810** [comment]
Looks good!

**Pull request merged**